spec/javascripts/services/short_form_data_service_spec.coffee

```diff
@@ -37,7 +38,24 @@
       ShortFormDataService = _ShortFormDataService_
       ListingDataService = _ListingDataService_
       ListingConstantsService = _ListingConstantsService_
-      ListingDataService.listing.preferences = getJSONFixture('listings-api-listing-preferences.json').preferences
+
+
+      fakePreferences = getJSONFixture('listings-api-listing-preferences.json').preferences
+      # Add a fake custom preference to the automated test listing.
+      fakePreferences = fakePreferences.concat([{
```

Comment:
Had to update these tests because I had originally manually edited one of the json fixtures to get it to work, which was wiped out by running the rspec tests.